jupyterlab_git/tests/test_clone.py

```diff
@@ -32,6 +32,32 @@ async def test_git_clone_success():
             assert {"code": 0, "message": output} == actual_response
 
 
+# Create a temporary directory and fake the content of it to be a directory containing a `.git` folder (that should be removed)
+@pytest.mark.asyncio
+@pytest.fixture
```

Comment:
This decorator is to mention a function is a test fixture. If you want to use a test fixture, you only need to request them as input argument of the test function

```suggestion
```

> The consequence is that your test was actually not executed. Otherwise you would have got some name clash with the previous test.